### PR TITLE
Amex fix

### PR DESCRIPTION
**Pull request by charmarkk**
Added rule for *.americanexpress.com, properly inverted dropdown select elements for payment bank accounts

### CODE REVIEW

**Comment by Myshor** [comment]
I pressume it's somewhere after loggin in.

Can you share some before and after fix screens? (of course blurred/censored to not share what is private).

Just to be sure what we fix here. 😉

**Comment by Myshor** [comment]
Thanks for screens and CSS preview. I will add a comment in review then. 😉

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -113,6 +113,15 @@ IGNORE INLINE STYLE
 
 ================================
 
+*.americanexpress.com
+
+CSS
+select.css-1ui3puo > option {
```
Comment (Myshor) [inline src/config/dynamic-theme-fixes.config:119]: change to:
`select[aria-labelledby="bankAccount-label"] > option`

this should be better than class which can be generated name and can change in the future. 😉

**Review by Myshor — CHANGES_REQUESTED** [review]

**Comment by charmarkk** [comment]
Fair enough! Working in accessibility we try not to use ARIA as a selector
for the same reason, but that makes sense!

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (charmarkk) [reply src/config/dynamic-theme-fixes.config:119]: Will go in and change later today.

Comment (charmarkk) [reply src/config/dynamic-theme-fixes.config]: Sorted,

**Review by Myshor — APPROVED** [review]

**Pull request merged**